### PR TITLE
Add Mailchimp Transactional API key to terraform secrets

### PR DESCRIPTION
**Pull request by liam-lloyd**
Now that we're using Mailchimp Transactional in this project, terraform needs to know about its API key so it can supply it to the deployments. This commit adds the API key to terraform's secrets.

### CODE REVIEW

File: terraform/test_cluster/secrets.tf [hunk 1]
```diff
@@ -17,9 +18,9 @@ resource "kubernetes_secret" "staging-secrets" {
...
 
   data = {
-    "DATABASE_URL"                 = var.staging_database_url
-    "FUSIONAUTH_API_KEY"           = var.staging_fusionauth_api_key
-    "LEGACY_BACKEND_SHARED_SECRET" = var.staging_legacy_backend_shared_secret
-    "MAILCHIMP_API_KEY"            = var.mailchimp_api_key
+    "DATABASE_URL"                    = var.staging_database_url
+    "FUSIONAUTH_API_KEY"              = var.staging_fusionauth_api_key
+    "LEGACY_BACKEND_SHARED_SECRET"    = var.staging_legacy_backend_shared_secret
+    "MAILCHIMP_TRANSACTIONAL_API_KEY" = var.mailchimp_transactional_api_key
```
Comment (cecilia-donnelly) [inline terraform/test_cluster/secrets.tf:24]: Elsewhere we have both the mailchimp API key and the transactional API key - should we have both here too?

**Review by cecilia-donnelly — COMMENTED** [review]
Thank you! Sorry I didn't think of this. One question but generally makes sense.

File: terraform/test_cluster/secrets.tf (hunk 1 above)
Comment (liam-lloyd) [reply terraform/test_cluster/secrets.tf:24]: Yes, we should, didn't realize the marketing key wasn't there yet

Comment (liam-lloyd) [reply terraform/test_cluster/secrets.tf:24]: Oh, it was there and I accidentally replaced it. Good catch!

**Review by cecilia-donnelly — APPROVED** [review]

**Pull request merged**